classification/train.py

```diff
@@ -226,14 +235,14 @@ def valid(model: Any,
     })
 
     print(f"Validation Loss:\t{running_loss/len(data_loader)} \n Validation mAP:\t{valid_map}" )
-    if valid_map > best_valid_map:
-        path = os.path.join("models",wandb_run.name + '.pt')
+    if valid_map > best_valid_map and wandb.run is not None:
```

Comment:
should we add an option for saving if people aren't running it with wandb? for people looking to use this package or us in the future for testing

Reply:
This got fixed in the base branch so I'll just fix it in a merge commit